roles/download/defaults/main.yml

```diff
@@ -39,7 +39,8 @@ flannel_server_helper_image_tag: "{{ flannel_server_helper_version }}"
 flannel_image_repo: "quay.io/coreos/flannel"
 flannel_image_tag: "{{ flannel_version }}"
 calicoctl_image_repo: "calico/ctl"
-calicoctl_image_tag: "{{ calico_version }}"
+# TODO(mattymo): v1.0.0-beta is not released yet
+calicoctl_image_tag: "v0.22.0"
```

Comment:
Actually v1.0.0-beta isn't compatible. It has different syntax. I'll change the comment